docs/user-guide/secrets/index.md

```diff
@@ -344,6 +344,9 @@ In this case, the file resulting in `/etc/foo/my-group/my-username` will have
 permission value of `0777`. Owing to JSON limitations, you must specify the mode
 in decimal notation.
 
+Also, take into account that this number might be displayed in decimal notation
```

Comment:
Suggested edit: "Note that this permission value might be displayed in decimal notation if you read it later."